src/main/java/org/jitsi/jicofo/Participant.java

```diff
@@ -284,6 +275,9 @@ public void setSupportedFeatures(List<String> supportedFeatures)
     {
         this.supportedFeatures
             = Objects.requireNonNull(supportedFeatures, "supportedFeatures");
+        if (!hasBundleSupport()) {
+            throw new IllegalStateException("Participant doesn't support bundle, which is required");
```

Comment:
is this exception handled anywhere?

Reply:
No, the idea was to cause a failure...I originally had it as a log but thought it would be better to fail here to catch anything.  Do you think I should change it back to a log?

Reply:
without  reading the entire PR I think 'hasBundleSupport()' should be called before starting the process of inviting a participant and  log an error or something

Reply:
My thinking was that we need to validate this after hearing back from the client, since we want to validate the client told us it supports bundle?

Reply:
can you just try to throw this exception there regardless and see what happens? if jicofo would catch it gracefully somewhere and log then it's fine, otherwise it should be handled gracefully by checking and logging I think